contracts/strategies/CurveStrategy.sol

```diff
@@ -505,14 +504,14 @@ contract CurveStrategy is Ownable, IStrategy {
         if (!disableRewardTokenClaim) {
             if (poolType == DEPOSIT_ZAP || poolType == LENDING_POOL) {
                 for (uint256 i = 0; i < numRewards; ) {
-                    amounts[i] = gauge.integrate_fraction(address(this)) - gaugeMinter.minted(address(this), address(gauge));
+                    amounts[i] = gauge.claimable_tokens(address(this)) + _rewardTokens[i].balanceOf(address(this));
```

Comment:
is this really intended? why are we replacing the functions previously used? just want to understand the context. thanks

Reply:
yeah this curve method uses the same logic but it is not  a `view` so better IMO to directly use this